packages/paste-core/components/pagination/src/PaginationItems.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {ULStyles, LIStyles} from './styles';
+import type {PaginationItemsProps} from './types';
+import {PaginationItemsPropTypes} from './proptypes';
+
+const PaginationItems = React.forwardRef<HTMLUListElement, PaginationItemsProps>(({children, ...props}, ref) => {
+  const [validChildren] = React.useMemo(
+    () => [
+      React.Children.toArray(children).filter((child) => React.isValidElement(child) || typeof child === 'string'),
+    ],
+    [children]
+  );
```

Comment:
How much of this do you thing we _really_ need?

Reply:
We could cut the ` || typeof child === 'string'` part because we'd only want consumers to use `PaginationArrow`, `PaginationNumbers`, or `PaginationNumber` here.